docs/rules/no-named-import.md

```diff
@@ -0,0 +1,20 @@
+# Prevent named imports from react (react/no-named-import)
+
+Prevent the use of named imports from React, so it's visible in code that it's React and not custom code. This rule
```

Comment:
What about hooks, like `useEffect` etc?

Reply:
I will add an options object which allows the developer to whitelist imports

Reply:
I've added an option (`forceImport`, which could be true/false), where the linter will report when `React.useState` is used for example and should be imported. This way the rule can force certain imports which were used (`React.useState`) but not imported (`import React from 'react'`).